vespajlib/src/main/java/com/yahoo/slime/SlimeInserter.java

```diff
@@ -5,7 +5,7 @@
  * Helper class for inserting values into a Slime object.
  * For justification read Inserter documentation.
  **/
-final class SlimeInserter implements Inserter {
+public final class SlimeInserter implements Inserter {
```

Comment:
see comments for ArrayInserter (prefer explicit ctor only)

Reply:
Done